span_test.go

```diff
@@ -242,3 +243,62 @@ func TestSpanLifecycle(t *testing.T) {
 	sp1.Release() // Now we will kill the object and return it in the pool
 	assert.True(t, sp1.tracer == nil, "span must be released")
 }
+
+func TestSpan_Logs(t *testing.T) {
+	tests := []struct {
+		logs []opentracing.LogRecord
+		name string
+		want []opentracing.LogRecord
+	}{
+		{
+			name: "if span logs is return nil",
+		},
+		{
+			name: "if span logs is zero len slice return nil",
+			logs: []opentracing.LogRecord{},
+		},
+		{
+			name: "if span logs len more than 0 return a copy",
+			logs: []opentracing.LogRecord{{}},
+			want: []opentracing.LogRecord{{}},
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			s := &Span{logs: tt.logs}
+			if got := s.Logs(); !reflect.DeepEqual(got, tt.want) {
```

Comment:
I think `assert.Equals(t, tt.want, s.Logs())` will achieve the same with better output